spec/rails_helper.rb

```diff
@@ -32,6 +45,8 @@
   # config.mock_with :flexmock
   # config.mock_with :rr
 
+  config.include WaitForAjax, type: :feature
+
```

Comment:
Yay!
On Sun, Mar 27, 2016 at 6:16 PM Nathan Stitt <notifications@github.com>
wrote:

> In spec/rails_helper.rb
> <https://github.com/openstax/accounts/pull/295#discussion_r57538235>:
>
> > @@ -32,6 +45,8 @@
> >    # config.mock_with :flexmock
> >    # config.mock_with :rr
> >
> > +  config.include WaitForAjax, type: :feature
> > +
>
> I've had good success triggering the error by running rspec with a set
> seed:
> bundle exec rspec ./spec/features/user_updates_password_spec.rb --seed
> 15515
>
> I also attempted to install a global after(:each) by adding:
>
>   config.after(:each, type: :feature, js: true) {
>     puts 'AFTER EACH'
>     wait_for_ajax
>   }
>
> The blog is executed, but the error still persists. However when I add the
> wait_for_ajax call to the the spec directly it seems to suppress the
> error.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openstax/accounts/pull/295/files/2305bbf73d0ce0f0625d55023fb6ec75304ff925#r57538235>
>